gulpfile.js

```diff
@@ -0,0 +1,40 @@
+const gulp = require('gulp');
```

Comment:
but it is not just samples - this is/will be part of the build process. This is how we build the library for browsers (i.e. as a single file).